docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
```

Comment:
Is it a permission or privilege?
The file name says permission and it says privilege. 
Make it consistent. 
It is described in the guide. Please don't create a PR that doesn't follow it.
https://github.com/scalar-labs/team-handbook/blob/master/engineering/guidelines/commit.md#pr-policy

Reply:
Sorry, It is Permission. Changed heading in c601da6